PMM-12712 Doc about new shards collector. (#778)
percona/mongodb_exporter: README.md

|1| The profiler collects data for operations that take longer than the value of `slowms` or that match a filter.<br> When a filter is set: <ul><li> The `slowms` and `sampleRate` options are not used for profiling.</li><li>The profiler only captures operations that match the filter.</li></ul>
|2|The profiler collects data for all operations.|

#### Enabling shards metrics gathering

> When is enabled by the `--collector.shards`, the exporter will expose all new metrics about sharded Mongo. 

Reviewer: ```suggestion
When shard metrics collection is enabled by the `--collector.shards`, the exporter will expose metrics related to sharded Mongo. 
```

Example, if shards collector is enabled:
```
# HELP mongodb_shards_collection_chunks_count sharded collection chunks.
# TYPE mongodb_shards_collection_chunks_count counter
mongodb_shards_collection_chunks_count{collection="system.sessions",database="config",shard="rs1"} 250
mongodb_shards_collection_chunks_count{collection="system.sessions",database="config",shard="rs2"} 250
```

> You can see shards names, it's collection, database and count.

Reviewer: ```suggestion
You can see shard name, it's collection, database and count.
```
